app/models/token.rb

```diff
@@ -10,39 +10,81 @@ class Token
 
   validate :token_validation
 
+  attr_reader :jwt_config
+
+  GC_METADATA_NAMESPACE = "https://goodcity.hk/jwt/metadata"
+
+  module Types
+    API = "api".freeze      # An 'api' token is used as a mean of resolving the current user when connecting to the REST API
```

Comment:
You can remove the `.freeze` from all string literals
and add magik comment at top level
`#frozen_string_literal: true`

Reply:
Having code functionalities in comments is a ticking time bomb imo... no clue why this was added in ruby 🤔